feat: install transifex client with req

aht007 opened the pull request.

aht007 committed:
feat: install transifex client with req

ormsbee commented:
Why is this changing the license for the entire project? That's a MAJOR decision, and unrelated to transifex.

ormsbee commented:
Also, please don't check in a binary.

aht007 commented:
@ormsbee looks like I forgot to close this PR as unnecessary. The change has already been merged and there were no changes related to license and binary file. I had added tx to gitignore and also reverted the license file to original content after installation step. I will close this PR once back at my workstation and after taking another look at this
